#!/usr/bin/env python

PACKAGE = "wave_sim_config"

from dynamic_reconfigure.parameter_generator_catkin import *

import math

gen = ParameterGenerator()

# Parameters for wind driven waves
gen.add("wind_angle", double_t, 0, "Wind angle [rad]", 0.0, 0.0, 2 * math.pi)
gen.add("wind_speed", double_t, 0, "Wind speed [m/s]", 0.001, 0.001, 100.0)

# Generate the config
exit(gen.generate(PACKAGE, "wave_sim_config", "WaveWind"))
